ggshield/core/config/user_config.py

```diff
@@ -61,6 +63,22 @@ def add_ignored_match(self, secret: IgnoredMatch) -> None:
                 return
         self.ignored_matches.append(secret)
 
+    def dump_for_monitoring(self) -> str:
+        return json.dumps(
+            {
+                "show_secrets": self.show_secrets,
+                "ignored_detectors_count": len(self.ignored_detectors),
+                "ignored_matches_count": len(self.ignored_matches),
+                "ignored_paths_count": len(self.ignored_paths),
+                "ignore_known_secrets_secrets": self.show_secrets,
```

Comment:
I think this should be:
```suggestion
                "show_secrets": self.show_secrets,
```